### PR TITLE
Buildbot config tidying

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: master.cfg [hunk 1]
```diff
@@ -226,16 +226,18 @@ if tcmalloc_slaves:
...
 
 doc_slaves = slaves.GetSlaves(config_helper.GenerateDoc)
-c['builders'].append(
-    BuilderConfig(name="doxygen-doc-generator",
-      slavenames=[s.name() for s in doc_slaves],
-      factory=doc_factory))
+if doc_slaves:
+  c['builders'].append(
+      BuilderConfig(name="doxygen-doc-generator",
+        slavenames=[s.name() for s in doc_slaves],
```
Comment (nomis52) [inline master.cfg]: Should probably fix the indent here next time (slavenames should be aligned under name)

**Comment by nomis52** [comment]
LGTM

File: master.cfg (hunk 1 above)
Comment (peternewman) [inline master.cfg]: Done.

**Comment by nomis52** [comment]
LGTM. Merge when ready.

**Pull request merged**